src/test/java/uk/nhs/digital/nhsconnect/lab/results/fixtures/FhirFixtures.java

```diff
@@ -29,7 +29,7 @@ public static Practitioner generateRequester(final String name, final Administra
         return requester;
     }
 
-    public static Bundle generateBundle() {
+    public static Bundle generateBundle(final Practitioner requester) {
```

Comment:
So when we have more things mapped, are we expecting this method's parameter list to grow?